### PR TITLE
Add Complete Actix 2.x REST Server to the community section of README

### PR DESCRIPTION
**Pull request by ddimaria**
I've spent several months creating a full-featured implementation and recently converted it to Actix 2.0.  I think this could help answer a lot of questions on how to implement Actix in a production/enterprise setup.  The description in the README is long, but wanted to show "what you get".   Let me know if you want this edited.  Thanks for considering!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,17 @@
...
+- [Roseline](https://github.com/DoumanAsh/roseline.rs) : A personal web site and discord & IRC bot to access simple SQLite database. Demonstrates usage of various actix and actix-web concepts.
+- [Actix Auth Server](https://hgill.io/posts/auth-microservice-rust-actix-web-diesel-complete-tutorial-part-1/) : Auth web micro-service with rust using actix-web - complete tutorial. See code in [examples/simple-auth-server](https://github.com/actix/examples/tree/master/simple-auth-server)
+- [actix-raft](https://github.com/railgun-rs/actix-raft) : An implementation of the Raft consensus protocol using the actix Actor framework.
+- [lemmy](https://github.com/dessalines/lemmy) : A federated alternative to reddit in rust.
+- [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix.
+- [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
+- [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
+- [Rust, Docker & GraphQL](https://github.com/jayy-lmao/rust-graphql-docker): An example of using Dataloaders, context, and a minimal docker container.
```
Comment (Jonathas-Conceicao) [inline README.md]: I think we shouldn't change the rest of the markdown file so we can keep the git history cleanner.

File: README.md [hunk 2]
```diff
@@ -3,15 +3,17 @@
...
+- [Complete Actix 2.x REST Server](https://github.com/ddimaria/rust-actix-example): Actix 2.x HTTP Server, Multi-Database Support (CockroachDB, Postgres, MySQL, Sqlite), JWT Support, Async Caching Layer with a Simple API, Public and Secure Static File Service, Diesel Database Operations are Non-Blocking, Filesystem Organized for Scale, .env for Local Development, Integrated Application State with a Simple API, Lazy Static Config struct, Built-in Healthcheck (includes cargo version info), Listeners configured for TDD, Custom Errors and HTTP Payload/Json Validation, Secure Argon2i Password Hashing, CORS Support, Unit and Integration Tests, Test Coverage Reports, Dockerfile for Running the Server in a Container, and TravisCI Integration.
```
Comment (Jonathas-Conceicao) [inline README.md]: I think this description should be shorter. Maybe just call it something like 'feature-full' or 'production-like' example and add some more general ideal of what it offers, like DB support and so on. People looking for examples could get more details when checking the linked repo.

File: README.md (hunk 1 above)
Comment (ddimaria) [reply README.md]: Yea, sorry, auto-format on save. I'll turn it off and revert.

Comment (ddimaria) [reply README.md]: Reverted

File: README.md (hunk 2 above)
Comment (ddimaria) [reply README.md]: I significantly shortened it.  Let me know if you want it shorter.  Thanks!

File: README.md [hunk 3]
```diff
@@ -12,7 +12,8 @@ A curated list of examples related to actix.
 * [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
 * [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
 * [Rust, Docker & GraphQL](https://github.com/jayy-lmao/rust-graphql-docker): An example of using Dataloaders, context, and a minimal docker container. 
+* [Complete Actix 2.x REST Server](https://github.com/ddimaria/rust-actix-example): Actix 2.x HTTP Server featuring multi-database support, auth/JWTs, caching, static files, app state, tests, coverage, and docker.
 
 ## Contribute
 
-Welcome to contribute !
+Welcome to contribute !
```
Comment (JohnTitor) [inline README.md]: Could you also drop this change? The other part looks good.

Comment (ddimaria) [reply README.md]: This is done

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**